sites/upsun/src/manage-resources/adjust-resources.md

```diff
@@ -60,6 +67,41 @@ you may need to set the resources again.
 After you've set resources, your environment is redeployed,
 which causes a short downtime.
 
+- **Manual commands:**
+
+Use the following CLI flags:
```

Comment:
I find the word flags a bit confusing.
I would use options instead, and maybe a sentence like « Use the following options with the resources:set command: ».
And same in the table below.

But it depends if we are used to use the term flag elsewhere to describe options.

But this is a minor detail, otherwise this whole PR looks great to me.
Thanks.